pyproject.toml

```
@@ -51,8 +50,11 @@ documentation = "https://www.blosc.org/python-blosc2/python-blosc2.html"
 # Skip unsupported python versions as well as 32-bit platforms, which are not supported anymore.
 skip = "cp36-* cp37-* pp37-* cp38-* pp38-* cp39-* pp39-* *-manylinux_i686  cp*-win32 *_ppc64le *_s390x"
```

```suggestion
skip = "*-manylinux_i686  cp*-win32 *_ppc64le *_s390x"
```
The python specific parts are unnecessary, they are already handled by the `requires-python` part.